databricks_cli/configure/cli.py

```diff
@@ -84,29 +108,41 @@ def _configure_cli_password(profile, insecure):
 
 @click.command(context_settings=CONTEXT_SETTINGS,
                short_help='Configures host and authentication info for the CLI.')
-@click.option('--token', show_default=True, is_flag=True, default=False)
+@click.option('--token', '-t', 'token', show_default=True, is_flag=True, default=False)
+@click.option('--token-file', '-f', 'token_file', default=None,
+              help='Instead of reading the token from stdin, ' +
+                   'read the token from a file provided by a secret store.')
+@click.option('--host', show_default=True, default=None,
+              help='Host to connect to.')
 @click.option('--aad-token', show_default=True, is_flag=True, default=False)
-@click.option('--insecure', show_default=True, is_flag=True, default=None)
+@click.option('--insecure', show_default=True, is_flag=True, default=None,
+              help='DO NOT verify SSL Certificates')
 @debug_option
 @profile_option
-def configure_cli(token, aad_token, insecure):
+def configure_cli(token, aad_token, insecure, host, token_file):
     """
     Configures host and authentication info for the CLI.
     """
     profile = get_profile_from_context()
     insecure_str = str(insecure) if insecure is not None else None
+
     if token:
-        _configure_cli_token(profile, insecure_str)
+        _configure_cli_token(profile=profile, insecure=insecure_str, host=host)
+    elif token_file:
+        _configure_cli_token_file(profile=profile, insecure=insecure_str, host=host,
+                                  token_file=token_file)
+        _configure_cli_token(profile=profile, insecure=insecure_str, host=host)
```

Comment:
Is this maybe a bug? I don't understand why we want to call this here.

Reply:
oops. that's a merge bug thanks.